### ISSUE TITLE
It should be possible to link directly to items 

### ISSUE THREAD

**Issue by robertoostenveld** [bids-standard/bids-website#72]
@KirstieJane posted the "Meet the BIDS Steering Group" item. To facilitate dissemination, it should be possible to directly link to this, and have it rendered as separate page (including photo's).

**Comment by KirstieJane** [comment]
Haha - we overlapped in writting issues. I'll close #73 now :)

> At the moment, we can link to specific news items by adding the heading in lower case (and with - instead of spaces) but it isn't easy to get that link itself.
> 
> For example:
> 
> * https://bids.neuroimaging.io/news.html#meet-the-bids-steering-group
> * https://bids.neuroimaging.io/news.html#new-website
> * https://bids.neuroimaging.io/news.html#bids-steering-group-election-and-governance-results
> 
> Does anyone know of any css etc that can make it easy to get the link - for example by hovering over the heading itself?

**Comment by robertoostenveld** [comment]
Not using CSS, but using Jekyll/Liquid: just like we have "Read more ..." I can add a "Link to this post ..." line at the bottom of the post.

I also encountered problems with the relative path to images: when the news item is included in the news page, the URL (and hence the relative link to the images) is different than when the news item appears on its own page. I will fix that (for now) by making it an absolute path and moving the images to /assets/img/2019-12-31.

**Comment by robertoostenveld** [comment]
I merged some changes which make it easier to link. I also updated the headings to get successively smaller from h1, h2, ...

The improvement to the images is not directly visible since I also added the `<!--more-->` line to get an excerpt (which is on the news page) and there is no image in the excerpt. But if there were an image in the except, it would also work.  

The suggestion to use CSS or similar to make headings and subheadings linkable remains valuable, not only for news items, but also for other long pages.
